### PR TITLE
Add pagination to mail list view

### PR DESCRIPTION
**Pull request by matheus-fga**
Adding pagination options to user mailing lists on 'manage_subscriptions.html' template and ordering the mailing lists alphabetically.

Signed-off-by: Emilie Morais <emilie.mo@hotmail.com>
Signed-off-by: Gabriel Silva <gabriel93.silva@gmail.com>
Signed-off-by: Italo Paiva <italosensei@hotmail.com>
Signed-off-by: Matheus Silva <matheussilva.fga@gmail.com>

### CODE REVIEW

**Comment by matheus-fga** [comment]
Same feature as colab/colab-superarchives-plugin#12.

**Comment by matheus-fga** [comment]
@chaws

**Comment by chaws** [comment]
@matheus-fga Thanks for the PR! Let us just decide something first. That was already implemented here: https://softwarepublico.gov.br/gitlab/softwarepublico/softwarepublico/commit/de2da32dde93f346ba7b904e8c9d78d343688ff3

So we problably better wait till we're done with SPB then merge your changes. Same for here: https://github.com/colab/colab-superarchives-plugin/pull/12

@chocoelho , what are your thoughts?

**Comment by chocoelho** [comment]
@chaws, you do understand that that pagination was related to the
spb-theme, right? It's supposed to follow the design implemented in
Noosfero's spb-theme. I'll take a look at this PR and see the
difference. @matheus-fga, could please attach some screenshots, it'll be
easier for us to understand what you're trying to do before hand.


Em 10-05-2016 11:17, Charles Oliveira escreveu:

**Comment by chaws** [comment]
Yeapers! My concern is if that will make two paginations when this goes inside SPB's Colab

**Comment by chocoelho** [comment]
Absolutely not. The theming strategy is by overriding templates, so this
should not interfere on what we're doing there :)


Em 10-05-2016 12:00, Charles Oliveira escreveu:

**Comment by chaws** [comment]
Oh yeah! Just did the template thingy the other day, you're right! My bad =D

**Comment by italopaiva** [comment]
screenshot from 2016-05-17 16 38 14
screenshot from 2016-05-17 16 38 20

**Comment by italopaiva** [comment]
The pagination is in the footer of email panel. @chocoelho

**Comment by italopaiva** [comment]
@chaws, any update on the merge of this PR?

**Comment by chaws** [comment]
sorry for the delay! Merged now!

File: colab/accounts/templates/accounts/manage_subscriptions.html [hunk 1]
```diff
@@ -26,11 +26,29 @@ <h3 class="panel-title">{{ email }}</h3>
               </div>
             {% endfor %}
           </div>
+          <div class="panel-footer text-center">
```
Comment (chaws) [inline colab/accounts/templates/accounts/manage_subscriptions.html]: @matheus-fga , this part was moved to https://github.com/colab/colab/blob/detach_super_archives/colab/accounts/templates/accounts/subscription_lists.html, after @italopaiva added search by listname

**Pull request merged**